src/GraphQL.Tests/ServiceProviderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using GraphQL.StarWars.IoC;
+
+namespace GraphQL.Tests
+{
+    public static class ServiceProviderExtensions
+    {
+        public static T Resolve<T>(this IServiceProvider services)
```

Comment:
Changing `Resolve<T>` to `GetService<T>` will be a closer term to `IServiceProvider`